Controller/ApiController.php

```diff
@@ -62,47 +65,43 @@ public function document($apiName, $contentType, $ouuid)
         return $this->service->getDocument($apiName, $contentType, $ouuid)->getResponse();
     }
 
-    private function treatFormRequest(Request $request, string $apiName)
+    public function handleFormPostRequest(Request $request, string $apiName, string $contentType, ?string $ouuid, string $csrfId, string $validationTemplate, int $hashcashLevel, string $hashAlgo, bool $forceCreate = false)
```

Comment:
number of arguments should be reconsidered?  (General opinion, a good practice is maximum 4 parameters, possible to do more via Objects?)

Reply:
NO, those are route parameters